analyzers/tests/SonarAnalyzer.UnitTest/TestCases/DisablingRequestValidation.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
```

Comment:
_Rebase conflict:_ This file should not be here. It's already on master and was moved to `TestCases/Hotspots`